### PR TITLE
Initial changes for ms (Malay Language/Bahasa Melayu)

### PR DESCRIPTION
**Pull request by zubir2k**
Updated common yaml and change ms nativeName to Bahasa Melayu for consideration

### CODE REVIEW

File: sentences/ms/_common.yaml [hunk 1]
```diff
@@ -1,12 +1,60 @@
...
+      - "celsius"
+      - in: "c"
+        out: "celsius"
+      - "fahrenheit"
+      - in: "f"
+        out: "fahrenheit"
+expansion_rules:
+  name: "[the] {name}"
+  area: "[the] {area}"
+  what_is: "(apa itu | apakah | apa)"
```
Comment (balloob) [inline sentences/ms/_common.yaml:52]: This one does not wrap a slot name like `{name}` and so the key `what_is` should be updated to be in Malay.

File: sentences/ms/_common.yaml [hunk 2]
```diff
@@ -1,12 +1,60 @@
...
+    values:
+      - "celsius"
+      - in: "c"
+        out: "celsius"
+      - "fahrenheit"
+      - in: "f"
+        out: "fahrenheit"
+expansion_rules:
+  name: "[the] {name}"
+  area: "[the] {area}"
```
Comment (balloob) [inline sentences/ms/_common.yaml:51]: Does Malay use "the" ?

File: sentences/ms/_common.yaml (hunk 1 above)
Comment (balloob) [reply sentences/ms/_common.yaml:52]: (this applies to all expansion rules that don't wrap slots)

Comment (zubir2k) [reply sentences/ms/_common.yaml]: noted.. removing this for time being. thank you

File: sentences/ms/_common.yaml (hunk 2 above)
Comment (zubir2k) [reply sentences/ms/_common.yaml]: ya.. i only noticed that and have removed [the]. thank you

**Pull request merged**